dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -545,9 +546,9 @@ export const AdSlot = ({
 							'ad-slot--merchandising-high',
 						].join(' ')}
 						css={[
-							merchandisingAdStyles,
 							fluidAdStyles,
 							fluidFullWidthAdStyles,
+							merchandisingAdStyles,
```

Comment:
Follows the convention of the other ad slots, where slot-specific styles take precedent over general fluid styles